src/node/destination.rs

```diff
@@ -74,7 +74,7 @@ impl AudioNode for AudioDestinationNode {
         // OfflineAudioContext, then the channel count mode cannot be changed.
         // An InvalidStateError exception MUST be thrown for any attempt to change the value.
         assert!(
-            !self.registration.context().offline(),
+            !self.registration.context().offline() || v == ChannelCountMode::Explicit,
```

Comment:
Cannot find where it is explained in the spec, maybe add something as comment ?

Reply:
Good point, added. I also found a spec issue related to the channelCount. Will double check and raise an issue at the spec repo